modules/distr-tracing-otel-send-traces-and-metrics-to-otel-collector-with-sidecar.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * /distr_tracing/distr_tracing_otel/distr-tracing-otel-using.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-send-traces-and-metrics-to-otel-collector-with-sidecar_{context}"]
+= Sending traces and metrics to the OpenTelemetry Collector with sidecar injection
+
+You can set up sending telemetry data to an OpenTelemetryCollector instance with sidecar injection.
+
+The {OTELOperator} allows sidecar injection into Deployment workloads and automatic configuration of your instrumentation to send telemetry data to the OpenTelemetry Collector.
+
+.Prerequisites
+
+* The {TempoName} is installed and a TempoStack instance is deployed.
+
+* You have access to the cluster through the OpenShift web console or the OpenShift CLI (`oc`):
+
+** You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role.
+
+** An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
+
+** For {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Create an OpenShift project for the OpenTelemtry Collector.
++
+[source,yaml]
+----
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: observability
+----
+
+. Create a service account.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-sidecar
+  namespace: observability
+----
+
+. Grant permissions to the service account for the `k8sattributes` and `resourcedetection` processors.
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+- apiGroups: ["", "config.openshift.io"]
+  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-sidecar
+  namespace: observability
+roleRef:
+  kind: ClusterRole
+  name: otel-collector
+  apiGroup: rbac.authorization.k8s.io
+----
+
+. Deploy the OpenTelemetry Collector as a sidecar.
++
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: otel
+  namespace: observability
+spec:
+  mode: sidecar
+  config: |
```

Comment:
Forgot to reference to the serviceAccount created in the previous steps.
We are missing this

`  serviceAccount: otel-collector-sidecar`

Reply:
Thank you. Done.